spec/services/feeds/import_spec.rb

```diff
@@ -87,17 +86,17 @@
     end
 
     context "with an explicit set of users", vcr: { cassette_name: "feeds_import" } do
+      # TODO: We could probably improve these tests by parsing against the items in the feed rather than hardcoding
```

Comment:
I was weary of going down a rabbit hole and completely re-writing these tests by parsing the VCR file, so I converted them to a baseline Rspec version, but left a comment here as to how they could be improved!